x/ccv/provider/keeper/relay.go

```diff
@@ -162,6 +162,13 @@ func (k Keeper) EndBlockVSU(ctx sdk.Context) {
 	}
 }
 
+// BlocksUntilNextEpoch returns the number of blocks until the next epoch starts
+// Returns 0 if the current block is the last block of the epoch
+func (k Keeper) BlocksUntilNextEpoch(ctx sdk.Context) int64 {
+	epochLength := k.GetBlocksPerEpoch(ctx)
+	return int64(epochLength - ctx.BlockHeight()%epochLength)
```

Comment:
Maybe:
```suggestion
	return int64(epochLength - (ctx.BlockHeight()%epochLength))
```
so it's slightly easier to parse.